src/locale/misc/smartSearch/en.yml

```diff
@@ -13,6 +13,7 @@ filterTitles:
     campaign_participation: People who have participated in a campaign
     most_active: A selection of the most active people
     person_data: Search based on name, address or similar fields
+    person_tags: Search for people with specific tags
```

Comment:
The "**with** specific tags" obscures the fact that this selection is also used to find people who are _missing_ certain tags. Maybe "Search for people based on tags" instead?